.github/workflows/test-build-deploy.yml

```diff
@@ -94,29 +94,33 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v3
 
-      - name: Unlock git-crypt secrets
-        uses: sliteteam/github-action-git-crypt-unlock@8b1fa3ccc81e322c5c45fbab261eee46513fd3f8
-        env:
-          GIT_CRYPT_KEY: ${{ secrets.GIT_CRYPT_KEY }}
-
-      - name: Authenticate to the cluster
-        env:
-          KUBE_CERT: ${{ secrets.KUBE_STAGING_CERT }}
-          KUBE_TOKEN: ${{ secrets.KUBE_STAGING_TOKEN }}
-          KUBE_CLUSTER: ${{ secrets.KUBE_STAGING_CLUSTER }}
-          KUBE_NAMESPACE: ${{ secrets.KUBE_STAGING_NAMESPACE }}
-        run: |
-          echo "${KUBE_CERT}" > ca.crt
-          kubectl config set-cluster ${KUBE_CLUSTER} --certificate-authority=./ca.crt --server=https://${KUBE_CLUSTER}
-          kubectl config set-credentials deploy-user --token=${KUBE_TOKEN}
-          kubectl config set-context ${KUBE_CLUSTER} --cluster=${KUBE_CLUSTER} --user=deploy-user --namespace=${KUBE_NAMESPACE}
-          kubectl config use-context ${KUBE_CLUSTER}
+      - name: Deploy staging
+        uses: ./.github/actions/deploy
+        with:
+          environment-name: staging
+          git-crypt-key: ${{ secrets.GIT_CRYPT_KEY }}
+          ecr-url: ${{ secrets.ECR_URL }}
+          kube-cert: ${{ secrets.KUBE_STAGING_CERT }}
```

Comment:
@zheileman Can we talk secrets? I'm a bit confused how the constants are doing to work :)